aeron-driver/src/main/c/util/aeron_error.c

```diff
@@ -168,8 +168,14 @@ void aeron_set_windows_error()
         0,
         NULL);
 
-    aeron_set_err(errorId, messageBuffer);
-    free(messageBuffer);
+	if (result == 0)
```

Comment:
Formatting is a bit screwed here. Is there something missing?

Reply:
Sorry, I inserted tabs here instead of spaces, will fix.